README.md

```diff
@@ -163,19 +179,34 @@ capable stylesheet transformer.
 
 ### Supported Transformations
 
-| From ╲ To  | hOCR               | ALTO               | PAGEXML | FineReader | Plain Text         |
-| ---:       | ---                | ---                | ---     | ---        | ---                |
-| hOCR       | -                  | :white_check_mark: | -       | -          | :white_check_mark: |
-| ALTO       | :white_check_mark: | -                  | -       | -          | :white_check_mark: |
-| PAGE       | -                  | -                  | -       | -          | -                  |
-| FineReader | -                  | -                  | -       | -          | -                  |
+| From ╲ To           | hOCR | ALTO | PAGEXML | FineReader | Plain Text  | Google Cloud Vision |
+| ---:                | ---  | ---  | ---     | ---        | ---         | ---                 |
+| hOCR                | -    | ✓    | -       | -          | ✓           | -                   |
+| ALTO                | ✓    | ✓    | -       | -          | ✓           | -                   |
```

Comment:
The check mark for a transformation between ALTO and ALTO seems a typo. I suggest to use the equal sign `=` on the diagonal.